homeassistant/components/litterrobot/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
+  # Bronze
+  action-setup:
+    status: todo
+    comment: |
+      Action async_set_sleep_mode is currently setup in the vacuum platform
+  appropriate-polling:
+    status: done
+    comment: |
+      Primarily relies on push data, but polls every 5 minutes for missed updates
+  brands: done
+  common-modules:
+    status: todo
+    comment: |
+      hub.py should be renamed to coordinator.py and updated accordingly
```

Comment:
Another thing to note, I am not sure why the state is `bool` while it never gets used